src/components/PostList/PostList.jsx

```diff
@@ -1 +1,14 @@
-export const PostList = () => <>Put the list here</>;
+import React from 'react';
+import { PostInfo } from '../PostInfo/PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => {
+      return (
+        <React.Fragment key={post.id}>
+          <PostInfo post={post} />
+        </React.Fragment>
+      );
+    })}
```

Comment:
1.  I'm not sure that here is a good place for React.Fragment
2. Let's think about how we can remove `return` keyword here